docs/cmdline-opts/http3.d

```diff
@@ -13,10 +13,15 @@ Example: --http3 $URL
 Multi: mutex
 Experimental: yes
 ---
-Tells curl to use HTTP version 3 directly to the host and port number used in
-the URL. A normal HTTP/3 transaction will be done to a host and then get
-redirected via Alt-Svc, but this option allows a user to circumvent that when
-you know that the target speaks HTTP/3 on the given host and port.
+Tells curl to try HTTP/3 to the host used in the URL, but fallback to use
+earlier HTTP versions if the HTTP/3 connection establishement fails. HTTP/3
+can only be used for HTTPS and not for HTTP URLs.
```

Comment:
Will it be silently ignored then for `http:`?

Reply:
That's a good question. I believe it right now returns an error if asked to do 3 with HTTP... I think we can go with that for now.